util/cgiwifi.c

```diff
@@ -103,22 +109,64 @@ void ICACHE_FLASH_ATTR wifiScanDoneCb(void *arg, STATUS status) {
 	cgiWifiAps.scanInProgress=0;
 }
 
+#else
+
+void ICACHE_FLASH_ATTR wifiScanDoneCb() {
+	uint16_t num_aps = 16;
+	wifi_ap_record_t ap_records[16];
```

Comment:
is the size of ap_records[] connected to num_aps? If so you probably want to do ap_records[num_aps]

Reply:
Yes, my bad. Updated to use a #define further up